tasks/eth/ink-001-permissionless-proofs/VALIDATION.md

```diff
@@ -0,0 +1,46 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade transaction.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff
+are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state
+  changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain
+  Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Changes
```

Comment:
We should also mention the nonce increments and `approveHashes` changes in here. See a recent example of how we did this here: https://github.com/ethereum-optimism/superchain-ops/blob/1d221adbb3dc73facbe9b8e4831bd61d3b44e858/tasks/eth/018-granite-upgrade/VALIDATION.md#L101-L111